layouts/shortcodes/observability_pipelines/destination_settings/microsoft_sentinel.md

```diff
@@ -6,13 +6,13 @@ To set up the Microsoft Sentinel destination, you need to create a Workspace in
 1. [Create a Log Analytics Workspace][10165] in the workspace if you haven't already.
 1. In the Log Analytics Workspace, navigate to **Settings** > **Tables**.
     1. Click **+ Create**.
-    1. Define a custom table (for example, `Custom-MyLogs_CL`).
-        - **Notes**:<br>- For custom tables, the table name must start with `Custom-`. `CL` is automatically appended to the end of the table name. You need the table name to set up the Observability Pipelines Microsoft Sentinel destination.<br>- You can also use an Azure Table instead of a custom table.
+    1. Define a custom table (for example, `MyOPWLogs`).
+        - **Notes**:<br>- After the table is configured, the prefix `Custom-` and suffix `_CL` are automatically appended to the table name (for example, `Custom-MyOPWLogs_CL`). You need the full table name to set up the Observability Pipelines Microsoft Sentinel destination. The full table name can be found in the resource JSON of the DCR under `streamDeclarations`.<br>- You can also use an Azure Table instead of a custom table.
     1. Select **New Custom Log (DCR-based)**.
     1. Click **Create a new data collection rule** and select the DCE you created earlier.
     1. Click **Next**.
     1. Upload a sample JSON Log. For this example, the following JSON is used for the **Schema and Transformation**, where  `TimeGenerated` is required:
-        ```json
+        ```
```

Comment:
I will add `json` back in before merge. Removed because the red highlighting made it difficult to read. I wasn't able to figure out why the text after gets highlighted in red.